### ISSUE TITLE
I think there's some issues on the ugtr.py

### ISSUE THREAD

**Issue by JosephKKim** [fanyang587/UGTR#5]
1. After line #97, if prob_x should be interpreted as probability I think sigmoid function should be applied to prob_x as well as prob_out2.

2. I haven't run file train.py (since the training data is not given...), but if target is binary consist of only 1 and 0 (it seems like you are using torch.where and divide the target by 255.0 to make it binary, in train.py), KL divergence loss always gives negative number (Please refer to the pytorch document). 

I figured this issues while applying your method to other dataset, so I might be wrong.
Please reply to this issue.

Thanks. 

Joseph
